build/Targets/Versions.props

```diff
@@ -9,9 +9,11 @@
 
   <PropertyGroup>
     <!-- This is the assembly version of Roslyn from the .NET assembly perspective. It should only be revved during significant point releases. -->
-    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.6.0</RoslynAssemblyVersionBase>
+    <!-- PROTOTYPE(NullableReferenceTypes): Revert before merging to master. -->
+    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.6.1</RoslynAssemblyVersionBase>
```

Comment:
Note this would also be whatever the first patch release of Roslyn will be versioned. I'd go higher than that.

Reply:
42.42.42.42?